models/cohere.py

```diff
@@ -17,11 +17,21 @@ def coherence(con_caption: str, unc_caption: str, ocr_output: list[str]) -> str:
                 f"adhere to the following guidelines to ensure the quality and relevance of your output: Generalize "
                 f"Brand References: If the inputs do not specify a brand, avoid using specific brand names in your "
```

Comment:
When you start a new instruction, "Brand References," "Maintain Grammatical Integrity," and "Start Every Caption With," make these start on a new line. This helps anyone else looking at the code to see how everything is organized.